11ty/definitions/colonialism.md

```diff
@@ -1,5 +1,27 @@
 ---
-title: Colonialism
+title: colonialism
 slug: colonialism
-defined: false
+speech: noun
+excerpt: the act of enforcing one's political and ideological dominance in a country or region that is not of one's origin
+sub_terms:
+    - text: modern
+    full_title: modern colonialism
+    - text: settler
+    full_title: settler colonialism
+    - text: Western
+    full_title: Western colonialism
+defined: true
+reading:
+    - text: A brief context of colonialism
+      href: https://plato.stanford.edu/entries/colonialism/
```

Comment:
```suggestion
skip_in_table_of_content: true
defined: true
reading:
  - text: A brief context of colonialism
    href: https://plato.stanford.edu/entries/colonialism/
```

Because this term is technically a "sub-term" of another term, it gets skipped in the table of contents using this front matter line.